pyinteraph/path_analysis.py

```diff
@@ -290,7 +292,28 @@ def filter_graph(graph, node_threshold, edge_threshold):
         log.warning("No metapaths found.")
     return filterd_graph
 
-def get_metapath(graph, res_id, res_space, node_threshold, edge_threshold):
+def normalize_graph(graph, normalize):
+    """Takes in a graph where all weights ranged between 0 to 1 and 
+    returns a graph where all weights are normalized (divided by max
+    weight)
+    """
+    normalized_graph = nx.Graph()
+    # Get max weights for nodes and edges
+    max_edge = max([d["e_weight"] for u, v, d in graph.edges(data = True)])
+    max_node = max([d["n_weight"] for n, d in graph.nodes(data=True)])
+    
+    for u, v, d in graph.edges(data = True):
+        # Add nodes first
+        normalized_graph.add_node(u, n_weight = \
+                                    graph.nodes()[u]["n_weight"]/max_node)
+        normalized_graph.add_node(v, n_weight = \
+                                    graph.nodes()[v]["n_weight"]/max_node)
+        # Add edge
+        normalized_graph.add_edge(u, v, e_weight = d["e_weight"]/max_edge)
+    return normalized_graph
```

Comment:
Leave a blank line between the for loop and the return statement (for better readability).